Makefile

```diff
@@ -6,6 +6,13 @@ LDFLAGS ?= -ldflags=-X=github.com/aws/karpenter/pkg/utils/project.Version=$(shel
 GOFLAGS ?= -tags=$(CLOUD_PROVIDER) $(LDFLAGS)
 WITH_GOFLAGS = GOFLAGS="$(GOFLAGS)"
 
+# detect nice binary
```

Comment:
In the interest of keeping this Makefile manageable, I think it's fair to assume that you need nice installed if you want to run the benchmark.

Reply:
I'm fine removing it entirely, doubt CI has it and if you want it locally you can just run `nice -19 make benchmark`